docs/content/about-iota/iota-wallet/how-to/import.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: Detailed instructions on how to import an existing wallet into the IOTA Wallet, including using the balance finder and other tools.
+teams:
+   - iotaledger/tooling
+tags: [iota-wallet]
+---
+import PaginatorNavLink from '@theme/PaginatorNavLink';
+import Seed from '../../../_snippets/about-iota/iota-wallet/seed.md'
+
+# Importing an Existing Wallet
+
+To import an existing wallet into the IOTA Wallet, you can use your [mnemonic recovery phrase](./import/mnemonic.mdx), a [ledger device](./import/ledger.mdx) and more. 
+
+## Migrating from deprecated Wallet
```

Comment:
I dont know if its very clear to users that the balance finder should be part of their process

Reply:
Anything that is clear enough is good for me